let markdown = 
`&nbsp;
{row}
 {column width=3}
{image?fileName=updates%2Ejpg&align=None&responsive=true}
 {column}
 {column width=9} 
The academic investigators funded through the AMP-AD Knowledge Portal grants are expected to release their data rapidly and to share them broadly, prior to publication. Both raw and normalized forms of data will be released as they become available - with the normalization process documented.
* Dara Release = release of new data
* Data Update = update to previously released data
* A list of available data is available on the [**Browse Data**](https://www.synapse.org/#!Synapse:syn2580853/wiki/409853) page
* A list of analysis results is available on the [**Browse Analysis**](https://www.synapse.org/#!Synapse:syn2580853/wiki/409851) page
 {column}
{row}
&nbsp;
* *Due to the rapid release of data there may be instances where corrections are made post-release. In that case we will notify everyone who has downloaded data and post an update to this page*
* Follow the [AMP-AD Knowledge Portal Updates](https://www.synapse.org/#!Synapse:syn2580853/discussion/threadId=2206) discussion thread for emails about data releases and other updates
###May 2018 - Data Release
**Additional Data for Previously Released Study**
* The [ADMC_ADNI1](https://www.synapse.org/#!Synapse:syn5592519) study. This release adds [bile acid metabolomics](https://www.synapse.org/#!Synapse:syn12036813) to the previously released lipidomics and data from the p180 platform. All data in this study has been generated on the ADNI1 cohort and requires access approval through [ADNI](https://ida.loni.usc.edu/collaboration/access/appLicense.jsp;jsessionid=AA9AE7C88A14C311D6CAAAE86B808DF0)
###April 2018 - Data Update
* The [ROSMAP WGS data](https://www.synapse.org/#!Synapse:syn10901595): 16 samples were identified that are either sex mismatches or discordant with previous array-based genotype data and recommended to be excluded. Samples are listed in [WGS_sample_QC_info](https://www.synapse.org/#!Synapse:syn12178037). Additional QC files can be found in the [QC files](https://www.synapse.org/#!Synapse:syn121779960 folder
* The [MSBB study](https://www.synapse.org/#!Synapse:syn3159438): 55 RNAseq, 8 WES, 15 WGS samples are recommended to be excluded or remapped to the correct donor. Samples are listed in [MSBB_RNAseq.WES.WGS_sample_QC_info](https://www.synapse.org/#!Synapse:syn12178047)


###April 2018 - Data Release
**Additional Data for Previously Released Study**
* The [SUNYStrokeModel](https://www.synapse.org/#!Synapse:syn7181520).  Includes electrophysiological and behavioral data (Active Place Avoidance, Novel Object Recognition) in the following models: C57BL/6J - (WT), BKS.Cg-Dock7m+/+Leprdb/J, APP/PS1, J20, a/a Dock 7m+/Dock7m+, a/a Dock7m+/+Leprdb, where mice have been tested at 5 mo after a stroke or sham transient middle cerebral artery occlusion (tMCAo) procedure
###March 2018 - Data Release
**Additional Data for Previously Released Study**
* The [ROSMAP](https://www.synapse.org/#!Synapse:syn3219045) study. Metabolomics data from brain and serum. Includes[ Bile Acids and the AbsoluteIDQ p180 assay](https://www.synapse.org/#!Synapse:syn10235592)
###February 2018 - Data Release #2
**New Human Study**
* The [DiCAD](https://www.synapse.org/#!Synapse:syn9944859) study (Interdisciplinary Research to Understand the Interplay of Diabetes, Cerebrovascular disease and Alzheimer’s disease): A study of 200 community dwelling late middle-aged Hispanics without dementia from Northern Manhattan. Includes a questionnaires of general health, neuropsychological testing, fasting phlebotomy and oral glucose tolerance testing, brain Magnetic Resonance Imaging (MRI), and Brain Amyloid Positron Emission Tomography (PET).
###February 2018 - Data Release
**Additional Data for Previously Released Study**
* The [TAUAPPms](https://www.synapse.org/#!Synapse:syn3157182) study. Custom-designed Nanostring chips were used to validate retained intron events in 12- and 20 month old CRND8 transgenic and non-transgenic mice. See [probeset details and data](https://www.synapse.org/#!Synapse:syn11854439)
###February 2018 - Data Update
*  Individuals in the [ROSMAP WGS](https://www.synapse.org/#!Synapse:syn10901595) set were missing clinical data. The [ROSMAP clinical file](https://www.synapse.org/#!Synapse:syn3191087) has been updated. Please download a new version
###January 2018 - Data Release #2
Jointly called WGS data from the [MSBB](https://www.synapse.org/#!Synapse:syn10901600) study, the [MayoRNAseq](https://www.synapse.org/#!Synapse:syn10901601) study, and the [ROSMAP](https://www.synapse.org/#!Synapse:syn3219045) are now available. Data released in November 2017 was variants called within each study/
###January 2018 - Data Release
**The following new studies, additional data for existing  studies, and analysis results have been provided with this release**
&nbsp;
**New Model Study**
* The [TyrobpKO_AppPs1](https://www.synapse.org/#!Synapse:syn10378730) study. RNAseq from mice with one or both copies of the Tyrobp gene knocked-out crossed with [APP/PS1 Jax](https://www.jax.org/strain/004462) mice to obtain different combination of the Tyrobp and AppPS1 genotypes. This complements RNAseq data released previously from the [TyrobpKO](https://www.synapse.org/#!Synapse:syn7123287) study.
* The [iPSCAstrocytes](https://www.synapse.org/#!Synapse:syn10344967) study. RNAseq from astrocytes differentiated from human induced pluripotent stem cells. This complements RNAseq data released previously from the [iPSCMicroglia](https://www.synapse.org/#!Synapse:syn7203233) study.
&nbsp;
**Additional Data for Previously Released Studies**
* The [EmoryDrosophilaTau](https://www.synapse.org/#!Synapse:syn7274101) model study. This study has previously released proteomics data. This release provides RNAseq from the same samples.
* The [ROSMAP](https://www.synapse.org/#!Synapse:syn3219045) study. 
       * [Targeted Proteomics by Selected Reaction Monitoring (SRM)](https://www.synapse.org/#!Synapse:syn10468856).
       * [RNAseq generated from Microglia isolated from 10 ROSMAP individuals](https://www.synapse.org/#!Synapse:syn11468526). 
* The [CHDWB](https://www.synapse.org/#!Synapse:syn8272982). A mapping file between the cognitive and clinical assessments, and metabolomics data and gene expression data available in GEO.
&nbsp;
**Results from Analysis of Previously Released Data**
* The [TAUAPPms](https://www.synapse.org/#!Synapse:syn3157182) model study. [Protein validation of Abi3 and PLCG2 in the app model](https://www.synapse.org/#!Synapse:syn11539387).
###December 2017 - Data Update
Files from the MSBB and MSBB_ArrayTissuePanel studies that contain brain bank ids (or individual ids) have been replaced with new files where the brain bank id has been replaced by a randomly assigned identifier. It is very important that files with the original brain bank ids are destroyed. Therefore, please delete any file you have previously downloaded and download a new version of the file if needed. No other changes have been made to the files. 
&nbsp;
**MSBB**
* MSBB_clinical.csv (syn6101474)
* MSBB_RNAseq_covariates.csv (syn6100548)
* MSBB_WES_covariates.csv (syn6101472)
* AMP-AD_MSBB_WGS__sample_barcode_brainBankID.csv (syn11384608)
* MSBB_Proteomics_PFC_TRAITS.csv (syn6100412) (analysis file)
* MSBB_Proteomics_PFC_traits.network.csv (syn9637147) (network file) 
&nbsp;
**MSBB_ArrayTissuePanel**
* AMP-AD_MSBB_MSSM_array_ApoE_genotype.txt (syn4069444)
* AMP-AD_MSBB_MSSM_covariates_mRNA_AffymetrixU133AB.tsv (syn3205399)
* AMP-AD_MSBB_MSSM_AffymetrixArray_sample_key.txt (syn4552846)
---
###November 2017 - Data Update
* [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404). The following samples were from individuals that do not meet control criteria (Braak > 3.0). 1957_CER, 1950_CER, 1950_TCX, and 1925_TCX. The [CER](https://www.synapse.org/#!Synapse:syn5223705) and [TCX](https://www.synapse.org/#!Synapse:syn3817650) covariate files have been updated marking all information for these samples as NA, and the [CER](https://www.synapse.org/#!Synapse:syn6126119) and [TCX](https://www.synapse.org/#!Synapse:syn6126114) QC files have been update with these samples. Please download the new version of these files.
---
###November 2017 - Data Release
**The following data and analysis results have been provided with this release**
#####Consortium Studies
* The [rnaSeqReprocessing](https://www.synapse.org/#!Synapse:syn9702085) study. Transcript abundances estimates for the previoulsy released data generated from reprocessing of RNAseq generated from the MayoRNAseq, MSBB and ROSMAP studies
#####Individual Studies
**New Human Study**
* [MC-CAA](https://www.synapse.org/#!Synapse:syn9779506). RNAseq data from the Mayo Clinic AD-CAA RNA-Seq (MC-CAA) study. This study is designed to identify the transcriptomic signatures that are associated with Cerebral Amyloid Angiopathy (CAA) pathology in AD brains in a cohort of 75 individuals
&nbsp;
**Additional Data for Previously Released Studies**
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) study 
       * [Whole Genome Sequencing](https://www.synapse.org/#!Synapse:syn10901601) from 350 subjects
       * [Pre and post QC genotypes](https://www.synapse.org/#!Synapse:syn8650953) used in the imputed dataset provided in the May 2017 data release
* The [ROSMAP](https://www.synapse.org/#!Synapse:syn3219045) study
       * [Whole Genome Sequencing](https://www.synapse.org/#!Synapse:syn10901595) from 1200 subjects
* The [MSBB](https://www.synapse.org/#!Synapse:syn3159438) study
       * [Whole Genome Sequencing](https://www.synapse.org/#!Synapse:syn10901600) from 354 subjects
&nbsp;
**Results from Analysis of Previously Released Data**
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) study
       * [Preliminary_PolygenicScore_Analysis](https://www.synapse.org/#!Synapse:syn10821882) based on analysis of modules generated from weighted gene co-expression networks
---
###May 2017 - Data Release
**The following data, analysis results and experimental resources have been provided with this release**
#####Consortium Studies
* [The rnaSeqReprocessing](https://www.synapse.org/#!Synapse:syn9702085) study. A uniformly processed RNAseq dataset has been generated across the AMP-AD contributed ROSMAP, MSBB, and MayoRNAseq studies. This release contains bam and gene count files from the reprocessed data.
#####Individual Studies
**New Human Study**
* The [CHDWB](https://www.synapse.org/#!Synapse:syn8272982) study. Cognitive and clinical assessments, and metabolic analysis from a cohort of healthy employees from Emory University 
&nbsp;
**New Experimental Models**
* The [JAX_APPPS1](https://www.synapse.org/#!Synapse:syn9850001) study. RNAseq from a mouse model of early onset AD
* The [iPSCMicroglia](https://www.synapse.org/#!Synapse:syn7203233) study. RNAseq from human primary microglia, iPSC derived microglia, and polarized macrophages from 12 cell lines
* The [SY5Y_Emory](https://www.synapse.org/#!Synapse:syn8513284) study. Proteomics data from SY5Y cells overexpressing APP, TAU, and  snRNP70 
* The [APOE-TR](https://www.synapse.org/#!Synapse:syn8391648) study. Behavioral and biochemical data from human APOE-targeted replacement mouse models
&nbsp;
**Additional Data for Previously Released Studies**
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) study. 
       * [Proteomics data](https://www.synapse.org/#!Synapse:syn7431760) from the Temporal Cortex from 200 individuals
       * [Imputed Genotypes](https://www.synapse.org/#!Synapse:syn8650955) generated from the Illumina’s Human Omni 2.5 + Exome array
* The [ADMC_ADNI1](https://www.synapse.org/#!Synapse:syn5592519) study. Metabolomics data and pipeline scripts
* The [MSDM](https://www.synapse.org/#!Synapse:syn4587607) study. Additional RNAseq data from TREM2 Drosophila models
&nbsp;
**Results from Analysis of Previously Released Data**
* The [MSBB](https://www.synapse.org/#!Synapse:syn3159438) study
       * [WGCNA Network Analysis](https://www.synapse.org/#!Synapse:syn8495241) of the proteomics data
       * [MEGENA and WINA Network Analysis](https://www.synapse.org/#!Synapse:syn7123049) of the RNAseq data
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) study. [WGCNA Network Analysis Immune Module](https://www.synapse.org/#!Synapse:syn8720036) from Temporal Cortex RNAseq data
* The [Banner](https://www.synapse.org/#!Synapse:syn7170616) study.  [rWGCNA Network Analysis](https://www.synapse.org/#!Synapse:syn8600467) of the proteomics data
* The [U1-70_PrimaryCellCulture](https://www.synapse.org/#!Synapse:syn5820226) study. [WGCNA Network Analysis](https://www.synapse.org/#!Synapse:syn8391961) of the proteomics data
* The [TAUAPPms](https://www.synapse.org/#!Synapse:syn3157182) study. [WGCNA Network Analysis Immune Module](https://www.synapse.org/#!Synapse:syn8650075) from TgCRND8 and APPPS1 models RNAseq data
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) and [TAUAPPms](https://www.synapse.org/#!Synapse:syn3157182) studies. [Comparative transcriptomics](https://www.synapse.org/#!Synapse:syn9688647). 
&nbsp;
[**Experimental Resources**](https://www.synapse.org/#!Synapse:syn2580853/wiki/409845)
* Tau and APP mouse models used in the [TAUAPPms study](https://www.synapse.org/#!Synapse:syn3157182) are available to the research community. See information about the models and who to contact for access
* The Mount Sinai Drosophila Melanogaster Aβ42 Model used in the [MSDM](https://www.synapse.org/#!Synapse:syn4587607) study is being made available to the community. This is a line carrying human Aβ42 using GAL4-UAS system.
###December 2016 - Data Update
The [**MSBB**](https://www.synapse.org/#!Synapse:syn3159438) study. A final sample QC has been completed for the Whole Exome Sequencing and RNA sequencing. The following QC and corrections have been made
* [WES](https://www.synapse.org/#!Synapse:syn4645334)
       * A sample QC was done by an all vs. all genotype similarity comparison. This identified 14 samples that were found to be outliers for showing excessive sample relatedness with at least one sample. These samples were discarded from further study.
       * Plink genotypes were QCd by removing individuals with missing rate > 0.5. SNPs with MAF < 0.01, missing genotyping rate > 0.5, or HWE test P value < 0.001 were removed.
       * Bam files, Plink and a VCF file has been provided for the final dataset
&nbsp;
* [RNAseq](https://www.synapse.org/#!Synapse:syn3157743)
       * For RNA samples sequenced multiple times the fastq file which had the largest number of mapped reads and less than 5% rRNA mapped reads were selected. Variants were called on the RNA-seq data using GATK and gender information imputed for each sample using RNA expression and the WES data when possible. The heterozygous variants from the RNA-seq data were compared to the variants in the WES data, in order to assign each RNA-seq sample to its corresponding DNA sequence. These multiple layers of information were used to correct, when necessary, mislabeling. For RNA-seq samples with documented matching WES the sample was removed if the discordance rate between RNA-seq sample and its best corresponding exome sequence was more than 10%. This removed 394 samples from the dataset for a final set of 1030 samples from 301 individuals.
       * Bam files, raw gene counts and normalized counts have been provided for the final dataset
&nbsp;
* [Metadata](https://www.synapse.org/#!Synapse:syn7392158)
       * The clinical file and separate WES and RNAseq covariate files have been aggregated into a Metadata folder. The most recent version of the WES and RNAseq covariate files contain the final QCd dataset. In addition, the most recent version of the RNAseq covariate file corrects for mislabeled samples by mapping them to the correct individual ID. 
---
###December 2016 - Data Update
The [**MSBB**](https://www.synapse.org/#!Synapse:syn3159438) study. A final sample QC has been completed for the Whole Exome Sequencing and RNA sequencing. The following QC and corrections have been made
* [WES](https://www.synapse.org/#!Synapse:syn4645334)
       * A sample QC was done by an all vs. all genotype similarity comparison. This identified 14 samples that were found to be outliers for showing excessive sample relatedness with at least one sample. These samples were discarded from further study.
       * Plink genotypes were QCd by removing individuals with missing rate > 0.5. SNPs with MAF < 0.01, missing genotyping rate > 0.5, or HWE test P value < 0.001 were removed.
       * Bam files, Plink and a VCF file has been provided for the final dataset
&nbsp;
* [RNAseq](https://www.synapse.org/#!Synapse:syn3157743)
       * For RNA samples sequenced multiple times the fastq file which had the largest number of mapped reads and less than 5% rRNA mapped reads were selected. Variants were called on the RNA-seq data using GATK and gender information imputed for each sample using RNA expression and the WES data when possible. The heterozygous variants from the RNA-seq data were compared to the variants in the WES data, in order to assign each RNA-seq sample to its corresponding DNA sequence. These multiple layers of information were used to correct, when necessary, mislabeling. For RNA-seq samples with documented matching WES the sample was removed if the discordance rate between RNA-seq sample and its best corresponding exome sequence was more than 10%. This removed 394 samples from the dataset for a final set of 1030 samples from 301 individuals.
       * Bam files, raw gene counts and normalized counts have been provided for the final dataset
&nbsp;
* [Metadata](https://www.synapse.org/#!Synapse:syn7392158)
       * The clinical file and separate WES and RNAseq covariate files have been aggregated into a Metadata folder. The most recent version of the WES and RNAseq covariate files contain the final QCd dataset. In addition, the most recent version of the RNAseq covariate file corrects for mislabeled samples by mapping them to the correct individual ID. 
###November 2016 - Data Release
**The following data, analysis results and experimental resources have been provided with this release**
&nbsp;
**New Human Study**
* The [Banner](https://www.synapse.org/#!Synapse:syn7170616) study. Proteomics data from the Prefrontal Cortex of 201 post-mortem brain tissue samples from the Banner Sun Health Research Institute's Brain and Body Donation Program
&nbsp;
**New Experimental Models**
* The [BroadAstrom109](https://www.synapse.org/#!Synapse:syn7204912) study. Aβ peptide multiplex ELISA following target gene perturbation in human primary astrocytes, iPSC generated neurons
* The [EmoryDrosophilaTau](https://www.synapse.org/#!Synapse:syn7274101) study. Proteomics data from TAU transgenic *Drosophila melanogaster* strains
* The [SUNYStrokeModel](https://www.synapse.org/#!Synapse:syn7181520) study. Behavioral analysis and Electrophysiology data from a stroke model study on WT (C57BL/6J) and db/db mice
* The [TyrobpKO](https://www.synapse.org/#!Synapse:syn7123287) study. RNAseq data from the Dentate Gyrus and Prefrontal Cortex of a TYROBP KO mouse model
&nbsp;
**Additional Data for Previously Released Studies**
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) study. [RNAseq data](https://www.synapse.org/#!Synapse:syn7344223) from the Temporal Cortex of 41 individuals with Pathological Aging
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) study. Post Mortem Interval (PMI) variables for subjects with [Cerebellum RNAseq](https://www.synapse.org/#!Synapse:syn5223705) and [Temporal Cortex RNAseq](https://www.synapse.org/#!Synapse:syn3817650)
* The [ROSMAP](https://www.synapse.org/#!Synapse:syn3219045) study. [iDAT files](https://www.synapse.org/#!Synapse:syn7357283) for the methylation assay
&nbsp;
**Results from Analysis of Previously Released Data**
* The [ACT](https://www.synapse.org/#!Synapse:syn5759376) study. [WGCNA Network Analysis]( https://www.synapse.org/#!Synapse:syn7180271) of the proteomics data 
* The [BLSA](https://www.synapse.org/#!Synapse:syn3606086) study. [WGCNA Analysis](https://www.synapse.org/#!Synapse:syn7180257) of the proteomics data
* The [ROSMAP](https://www.synapse.org/#!Synapse:syn3219045) study. [SpeakEasy Network Analysis](https://www.synapse.org/#!Synapse:syn7169817) of the RNAseq data
* The [MSBB](https://www.synapse.org/#!Synapse:syn3159438) study. [MEGENA and WGCNA Network Analysis](https://www.synapse.org/#!Synapse:syn7123049) of the RNAseq data
* The [MSBB - Array Tissue Panel](https://www.synapse.org/#!Synapse:syn3157699) study. [WGCNA Network Analysis](https://www.synapse.org/#!Synapse:syn3157700) of the array expression data
* The [MayoRNAseq](https://www.synapse.org/#!Synapse:syn5550404) study. [Differential Expression Analysis](https://www.synapse.org/#!Synapse:syn7332090) of the Cerebellum RNAseq data
* The [TAUAPPms](https://www.synapse.org/#!Synapse:syn3157182) study. [Differential Expression Analysis](https://www.synapse.org/#!Synapse:syn7317567) of the APP model
&nbsp;
**Experimental Resources**
* [Recombinant Adeno Associated Virus Vectors (rAAV) tools](https://www.synapse.org/#!Synapse:syn7357091). An extensive library of rAAV tools has been made available to the community. The documentation includes an overview of the promoters, capsids, fluorescent proteins, organelle markers and neurodegenerative disease relevant proteins encoded in the various rAAV vectors and a list the various rAAV constructs that has been generated. 
---
###Fifth AMP-AD Data Release:  May 27 2016
* [Mount Sinai Brain Bank (MSBB) AD Cohort](www.synapse.org/#!Synapse:syn3159438): Includes the following
       * RNAseq and WES from additional samples. This concludes the RNAseq and WES submission from this study
       * Proteomics from the PFC
       * Coexpression Networks from three brain regions
* [Mayo RNAseq Study](www.synapse.org/#!Synapse:syn5550404):Differential gene expression data from the temporal cortex
* [Harvard/MIT SY5Y REST model](www.synapse.org/#!Synapse:syn5987497): ChIPseq from the human SH-SY5Y neuroblastoma cell line incubated in the absence or presence of LiCl 
* [Harvard/MIT HDAC1-cKO Brain model](www.synapse.org/#!Synapse:syn5987310): RNAseq from aged HDAC1 cKO mice created to define the HDAC1-regulated gene networks. In addition, HDAC1-regulated transcriptome changes were also assessed by crossing HDAC1 cKO mice with 5xFAD AD model
* [Mt Sinai iPSC Neurons and NPCs study](www.synapse.org/#!Synapse:syn5986884): RNAseq from hiPSC forebrain neurons and NPCs
* [Emory ACT study](www.synapse.org/#!Synapse:syn5759376): Proteomics data from the Adult Changes in Thought cohort
* [Emory Emory U1-70 primary cell culture model](www.synapse.org/#!Synapse:syn5820226): Proteomics data from rat primary neuronal cells transduced with lentivirus U1-70K constructs
* [GSK TASTPM study](www.synapse.org/#!Synapse:syn5714307): Array based RNA expression data from TASTPM, a double transgenic mouse model co-expressing the human amyloid precursor protein and the human presenilin-1 mutation
* [GSK TauD35 study](www.synapse.org/#!Synapse:syn5714308): RNAseq data from TauD35, a tauopathy model expressing human tau P301L 2N4R driven by CAMKII promoter in C57BL/6 J
###Fourth AMP-AD Data Release:  December 4 2015
#####!Newly Generated Data in Release 4
* [Mt Sinai Brain Bank RNA sequencing study](syn3157743) Raw data for whole exome sequencing  as well as  RNA sequencing from additional samples
* [Mayo Clinic Brain Expression Studies](syn2910255):  RNAseq data generated from cerebellum
* [ROS/MAP](syn3219045):  REST protein quantification dervied from prefrontal cortex and H3K9Ac ChIP-seq data derived from dorsolateral prefrontal cortex
* [ADNI](syn5517723):  Metabolomic data collected from blood samples
* [Emory ADRC Brain Bank](syn3218563):  Proteomic data collected from DLPFC
* [UPenn Brain Bank](syn5477237):  Proteomic data collected from DLPFC
* [Mt Sinai Mouse Model](syn5477220): RNA sequencing data generated from entorhinal cortex and dentate gyrus from APP and PSEN mouse models.
* [Mt Sinai Drosophila Model](syn4587607): Raw data from RNA sequencing of  transgenic fly lines previously released.
* [Lilly Primary Microglia](syn5343747): RNA sequencing data from microglia isolated from rTg4510 tau mouse model
#####!Existing (Preaward) Data in Release 4
*No existing data are involved in this release*
#####!Analyses in Release 4
*No analyses are involved in this release*
###Third AMP-AD Data Release:  August 28 2015
#####!Newly Generated Data in Release 3
* [Mt Sinai Brain Bank RNA sequencing study](#!Synapse:syn3157743) Whole exome sequencing on all individuals as well as additional RNA sequencing from additional samples
* [Baltimore Longitudinal Study of Aging](syn3606086): Proteomic data from precuneus homogenate
* [Mayo Clinic LOAD Risk Case Control Association Study](#!Synapse:syn2910256): Genotypes for protein altering variants in *TLR4* and *TLR5*
* [Mt Sinai Brian Bank Expression Array Study](https://www.synapse.org/#!Synapse:syn3157700): CEL files are now available.
* [Mt Sinai Drosophila Model](syn4587607): RNA sequencing data generated from single, double, and triple transgenic fly lines for TREM2, TYROBP, and AB42.
#####!Existing (Preaward) Data in Release 3
*No existing data are involved in this release*
#####!Analyses in Release 3
*No analyses are involved in this release*
###Second AMP-AD Data Release:  May 28 2015
#####!Newly Generated Data in Release 2
* [ROS/MAP](https://www.synapse.org/#!Synapse:syn3219045): RNA sequencing data generated from prefrontal cortex.
* [Baltimore Longitudinal Study of Aging](syn3606086): Proteomic data
* [Mayo Clinic Brain Expression Studies](https://www.synapse.org/#!Synapse:syn2910255):  RNAseq data generated from temporal cortex
* [Monocyte-derived microglia cells](syn3607404): RNAseq data generated from MDMi
* [APP and Tau Mouse Models](https://www.synapse.org/#!Synapse:syn3157182): RNA sequencing data generated from two APP mouse models. 
* [iPSC Model](syn3607401): RNA sequencing data generated from iPSC-derived neural progenitor cells and mature neurons.  Astrocytes isolated from human brain.
#####!Existing (Preaward) Data in Release 2
* [ROS/MAP](https://www.synapse.org/#!Synapse:syn3219045): Expression array and miRNA data derived from frontal cortex
* [Harvard Brain Bank](https://www.synapse.org/#!Synapse:syn3159435): Imputed genotype data
#####!Analyses in Release 2
*No analyses are involved in this release.*
###First AMP-AD Data Release: March 3 2015
This release was largely comprised of existing, pre-award data from human samples that was used in generating preliminary results on the grant applications of the U01 AMP-AD projects. Study descriptions are provided in detail in [The AMP-AD Data](https://www.synapse.org/#!Synapse:syn2580853/wiki/77552) section. These include:
#####!Newly Generated Data in Release 1
* [Mt Sinai Brian Bank RNAseq Study](https://www.synapse.org/#!Synapse:syn3159438): RNA sequencing and clinical data generated from 3 brain regions of human postmortem brain samples collected from AD and control individuals.  
* [APP and Tau Mouse Models](https://www.synapse.org/#!Synapse:syn3157182): RNA sequencing data generated from two Tau mouse models. 
#####!Existing (Preaward) Data in Release 1
* [ROS/MAP](https://www.synapse.org/#!Synapse:syn3219045): DNA methylation, genotype, clinical data
* [Mt Sinai Brain Bank Expression Array Study](https://www.synapse.org/#!Synapse:syn3157699): expression array, clinical data and co-expression networks
* [Mayo Clinic LOAD Risk Case Control Association Study](https://www.synapse.org/#!Synapse:syn2910256): genotype, covariate data
* [Mayo Clinic Brain Expression Studies](https://www.synapse.org/#!Synapse:syn2910255):  expression array, RNA sequencing, and covariate data (with genotypes available for these samples within the Mayo Association Study)
* [Emory ADRC Brain Bank](https://www.synapse.org/#!Synapse:syn3218563): pooled proteomic data
* [Harvard Brain Bank](https://www.synapse.org/#!Synapse:syn3159435): expression array, genotype, and covariate data
#####!Analyses in Release 1
* [Mt Sinai Brian Bank Expression Array Study](https://www.synapse.org/#!Synapse:syn3157700): Coexpression networks
* [Mayo Clinic Brain Expression Study](https://www.synapse.org/#!Synapse:syn3157248): eGWAS results using expression arrays
&nbsp;
&nbsp;
&nbsp;
---`

const addSpaceToHash = string => {
  for( var index = 0; index < string.length; index++){
    if(string[index] === '#' 
			&& string[index+1] !== '#' 
			&& string[index+1] !== ' '
		){
      string = string.slice( 0, index) + ` ` + string.slice(index + 1, string.length)
    }
  }
	return string;
}

const defaultWikiData = addSpaceToHash(markdown) 

export default defaultWikiData 
